app/models/friend_request.rb

```diff
@@ -0,0 +1,7 @@
+class FriendRequest < ApplicationRecord
+  belongs_to :requester, class_name: 'User'
+  belongs_to :requested, class_name: 'User'
+
+  validates :requester, presence: true
```

Comment:
We don't need to validate presence of requester and requested. The associations in lines 2-3 themselves already validate the presence of these two attributes.